grails-app/conf/application.yml

```diff
@@ -174,3 +174,9 @@ dashboard:
 csv:
   temp:
     dir: "/data/dashboard/csv/"
+
+useCitizenScienceService: true
```

Comment:
Feel free to default these to `false` either here or in Ansible, and ALA, as the main user, can switch them back on using ansible when deploying our dashboard instance.